packages/components/simple-table/simple-table.styles.js

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+
+const Cell = styled.div`
+  display: flex;
+  align-items: center;
+
+  ${props => {
+    if (props.alignment === 'center') {
+      return `
+      justify-content: center;
+      text-align: center
+      `;
+    }
+    if (props.alignment === 'right') {
+      return `
+      justify-content: flex-end;
+      text-align: right;
+      `;
+    }
+    return `
+      justify-content: flex-start;
+      text-align: left;
+      `;
+  }};
+
+  padding: ${props => (props.isCondensed ? '2px 4px' : '8px 16px')};
+`;
+
+const Grid = styled.div`
+  display: grid;
+  /* stylelint-disable function-whitespace-after */
+  grid-template-columns: ${props =>
+    props.columns.map(column => column.width || 'auto').join(' ')};
+  /* stylelint-enable function-whitespace-after */
+  border-top: 1px solid ${vars.colorNeutral90};
+  border-left: 1px solid ${vars.colorNeutral90};
+
+  ${Cell} {
+    border-right: 1px solid ${vars.colorNeutral90};
+    border-bottom: 1px solid ${vars.colorNeutral90};
+  }
+
+  ${props => (props.maxHeight ? `max-height: ${props.maxHeight}px;` : '')}
+  ${props =>
+    props.maxWidth ? `max-width: ${props.maxWidth}px;` : ''}
+
+  overflow: auto;
+`;
+
+const Header = styled.div`
```

Comment:
Would it make sense to just use th, td and in general semantic markup for the new table and not a divs? What lead to the decision of using divs in general?

Reply:
That is a good question, as I am an adept of using semantic markup whenever possible 😄
I started by using it at first, but after studying some approaches, I realized it has some limitations, the biggest one being that it can't be combined with CSS-grid (which enables a lot of really cool layout magic) unless by using the css rule `display: contents` [which is not very well supported by all browsers](https://caniuse.com/#search=display%20contents).

Also, I didn't find the semantic markup of tables to be particularly useful, in terms of a11y navigation and screen readers. We can add some aria attributes to ease job of screen readers.

Lastly, the current table also doesn't use semantic `table` markup, so I didn't consider this a critical requirement.

That being said, it is still an option on the table (pun intended 👀) and I will continue to study this possibility, but currently, the ⚖️ of "table markup" vs "css-grid" is tending towards the latter 😄.

Reply:
Do we plan to still have **one single** table?

If not, we can have different kind of tables, with e.g. one of the basic ones being built as an HTML table.

Can you clarify this a bit?

Reply:
I think we will have multiple tables. I assume from discussions and the name BasicTable here. Which ones specifically we need/want and how to split interactions/use cases across them we should dicuss if not already somewhere :)

Reply:
The plan is to have a few exported tables with different added functionalities (such as sorting), and perhaps also exported "cell types" that one could use for the `renderItem` for common use cases.

But the "base" table would be the same for all of them, and the main challenge is to keep its layout flexible and responsive (and this is where css-grid shines).

I will put a bit more focus on trying to use `table` markup. In fact, with my current challenge of hovering/selecting whole rows, having an element for a whole row can be quite useful (but it breaks the css grid), so I will test more thoroughly how the `display: contents` rule behaves on all major browsers to see if it could be usable.

Reply:
Cool, thanks for the explanation 💯